modules/virt-virtctl-commands.adoc

```diff
@@ -55,6 +55,12 @@ the specified port of the node.
 |`virtctl vnc <vmi_name>`
 |Open a VNC connection to a virtual machine instance.
 
+|`virtctl vnc --kubeconfig=$KUBECONFIG <vmi-name>`
```

Comment:
This is similar to `virtctl vnc <vmi_name>`, the `--kubeconfig=$KUBECONFIG` can added to it as well.
Don't think you need a separate example to mention it, maybe just edit the one on Line 55

Reply:
@vatsalparekh - Please review the edit/change that was made and provide your response.